core/include/tee/tee_svc.h

```diff
@@ -52,7 +52,15 @@ uint32_t syscall_dummy_7args(unsigned long a1, unsigned long a2,
 
 uint32_t syscall_nocall(void);
 
-TEE_Result syscall_get_property(unsigned long prop, void *buf, size_t blen);
+unsigned long syscall_get_number_properties(unsigned long propSet);
+TEE_Result syscall_get_property(unsigned long prop_set,
+				void *name, unsigned long name_len,
```

Comment:
I do not see how. From the user side, we can get a property from its name or from an index. As the property is now only-defined in the kernel, I think we need both to call it from the user-side. Where am I wrong?